plugins/experimental/inliner/cache-handler.h

```diff
@@ -326,11 +326,12 @@ namespace inliner
       request             += std::string(b1, i);
       request             += "\r\n\r\n";
 
-      ats::io::IO *const io = new io::IO();
+      std::unique_ptr<ats::io::IO> io{new io::IO()};
 
       TSDebug(PLUGIN_TAG, "request:\n%s", request.c_str());
```

Comment:
It's not clear what exactly Coverity's gripe is.  My best guess is that it's seeing that `*io` would get leaked if TSDebug() threw an exception.  But there must be a bazillion cases of that in our code, why would it single this one out?  Are there instructions anywhere about how to run Coverity, to be sure we've appeased it?

Reply:
No unfortunately we don’t have enough karma to run PRs against coverity. So it’ll have to land on master for the next schedule run, rinse repeats…

Reply:
It looks like from reading the defect report, coverity doesn't trace into the second get call to see that it is indeed assigned eventually.  The std::move will hopefully realize that ownership is handed off.